modules/cluster-logging-forwarding-separate-indices.adoc

```diff
@@ -0,0 +1,69 @@
+// Module is included in the following assemblies:
+//cluster-logging-external
+:_content-type: PROCEDURE
+[id="cluster-logging-forwarding-separate-indices_{context}"]
+= Forwarding JSON logs from containers in the same pod to separate indices
+
+[IMPORTANT]
+====
+JSON formatting of logs varies by application. Because creating too many indices impacts performance, limit your use of this feature to creating indices for logs that have incompatible JSON formats. Use queries to separate logs from different namespaces, or applications with compatible JSON formats.
+====
+
+You can forward structured logs from different containers within the same pod to different indices. To use this feature, you must configure the pipeline with multi-container support, and annotate the pods. Logs are written to indices with a prefix of `app--`. It is recommended that Elasticsearch be configured with aliases to accommodate this.  
+
+.Prerequisites
+
+* {logging-title-uc}: 5.5
+
+.Procedure
+. Create or edit a YAML file that defines the `ClusterLogForwarder` CR object:
++
+[source,YAML]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputDefaults:
+    elasticsearch:
+      enableStructuredContainerLogs: true <1>
+  pipelines:
+  - inputRefs:
+    - application
+    name: application-logs
+    outputRefs:
+    - default <2>
+    parse: json
+----
+<1> Setting the value `enableStructuredContainerLogs` to `true` enables multi-container outputs.
+<2> You can specify alternate outputs or leave the `default` value.
```

Comment:
As a user reading this, I'm not sure what to do. Is this related to configuring Elasticsearch with aliases? Could you please help me make the choice?